core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -101,7 +101,7 @@ final class ClassfileIndexer(file: FileObject) extends SLF4JLogging {
       super.visitField(access, name, desc, signature, value)
       new FieldVisitor(ASM5) with ReferenceInFieldHunter {
         override def visitEnd(): Unit = {
-          internalRefs.add(ClassName.fromDescriptor(desc))
+          internalRefs.add(FullyQualifiedReference(ClassName.fromDescriptor(desc), None))
```

Comment:
why are there so many `None` uses here? Surely we know the most recently seen line number?